packages/remix-dev/__tests__/replace-remix-imports-test.ts

```diff
@@ -1,10 +1,10 @@
+import os from "os";
 import path from "path";
 import fse from "fs-extra";
-import os from "os";
+import glob from "fast-glob";
+import shell from "shelljs";
 import stripAnsi from "strip-ansi";
 import type { PackageJson } from "type-fest";
-import shell from "shelljs";
-import glob from "fast-glob";
```

Comment:
nit: we don't have linting/formatting for import ordering intentionally and generally want to avoid `git` noise from rearranging imports, just to make the diff simpler and reviewing easier. No big deal and we can merge this fine, but just mentioning it for future changes.